lfs.c

```diff
@@ -4911,3 +4911,398 @@ int lfs_migrate(lfs_t *lfs, const struct lfs_config *cfg) {
 }
 
 #endif
+
+#if LFS_THREAD_SAFE
+
+int lfs_format_ts(lfs_t *lfs, const struct lfs_config *config)
+{
+    int err = config->lock(config);
+    if(err)
+    {
+        return err;
+    }
+    err = lfs_format_raw(lfs, config);
+    config->unlock(config);
+
+    return err;
+}
+int lfs_mount_ts(lfs_t *lfs, const struct lfs_config *config)
```

Comment:
nit: I don't think we need the `_ts` suffix. If the user specifies LFS_THREAD_SAFE, they should be able to assume all functions are thread safe.

Reply:
fixed